src/games/strategy/util/Tuple.java

```diff
@@ -17,13 +17,13 @@
    * This method allows for nicer tuple creation syntax, namely:
    *    Tuple<String,Integer> myTuple = Tuple.of("abc",123);
    * Instead of:
-   *    Tuple<String,Integer> myTuple = Tuple.of("abc",123);
+   *    Tuple<String,Integer> myTuple = new Tuple<String,Integer>("abc",123);
```

Comment:
The constructors of Tuple should still be private, so 'new Tuple' is actually not the way to get an instance. I'm unsure of this comment update.